### ISSUE TITLE
DOC: link geospatial-python course in the docs

### ISSUE THREAD

**Issue by raybellwaves** [corteva/rioxarray#279]
https://carpentries-incubator.github.io/geospatial-python/ is a good course that uses rioxarray. Could add a link to it in https://corteva.github.io/rioxarray/stable/getting_started/getting_started.html

**Comment by snowman2** [comment]
I would like the PROJ string stuff replaced by `pyproj.CRS` first since it is deprecated: https://github.com/carpentries-incubator/geospatial-python/issues/40

**Comment by snowman2** [comment]
Also related: https://www.earthdatascience.org/search/?q=rioxarray

**Comment by rbavery** [comment]
Good point, I'm on board. I'll update this issue later this week once the PROJ stuff has been replaced with pyproj.CRS.

**Comment by rbavery** [comment]
@snowman2 @raybellwaves This has been addressed now, we use pyproj for summarizing CRSs: https://carpentries-incubator.github.io/geospatial-python/05-raster-structure/index.html
